releases/0.8.0/index.md

```diff
@@ -18,14 +18,16 @@ custom_css: releases
 *(These are changes to ontologies, classes or properties in the preexisting ontology that make the new release non-backward-compatible.)*
 
 * Restructured repository by moving previously prefixed uco-* ontology files under new directory /ontology/* ([*Change Proposal 56*](https://drive.google.com/file/d/1PCjdCGw7wgFnPfbsCn0Fdvdnq9NDW6yA/view))
+* Remove ActionReferenceFacet and move properties into action:Action ([*Change Proposal 89*](https://drive.google.com/file/d/1njOUmpNNCTolnGkXoKv5ext-ZK18108u/view))
 
 #### Changes
 *(These are general changes to the preexisting ontology that are not breaking or range changes.)*
 
+* Remove observable:fileSystemType from FileFacet ([*Change Proposal 45*](https://drive.google.com/file/d/1ycbboFe8wEYzD3AD6lbUadwXXFuGF3Dl/view))
```

Comment:
This link is either not set to 'view all with link' or the document does not exist.